src/mach_dep.c

```diff
@@ -7,6 +7,15 @@
 #include	"rogue.h"
 #include	"curses.h"
 
+#ifndef ROGUE_NO_X11
+#include <X11/Xlib.h>
+
+/*@
+ * Pointer to X display, if running under X.  Used to query keyboard LED status.
+ */
+Display *xdisplay;
```

Comment:
Maybe `Display *xdisplay = NULL;` just to be safer?

Reply:
Good point, will do that.